R/apply_theme.R

```diff
@@ -20,18 +21,23 @@
 #'                                                         "Xanomeline High Dose" = "purple",
 #'                                                         "Xanomeline Low Dose" = "brown")),
 #'                             fontsizes = list("axis" = 12,
-#'                                              "ticks" = 10),
+#'                                              "ticks" = 10,
+#'                                              "legend_title" = 10,
+#'                                              "legend_text" = 8),
 #'                             fontfamily = "Helvetica",
-#'                             grid = FALSE,
-#'                             bg = "transparent")
+#'                             grid = list("major" = FALSE,
+#'                                         "minor" = FALSE),
+#'                             bg = "transparent",
+#'                             legend_position = "top")
 #'
 #' @export
 
 define_theme <- function(strata = NULL,
                          fontsizes = NULL,
                          fontfamily = "Helvetica",
                          grid = FALSE,
-                         bg = "transparent") {
+                         bg = "transparent",
+                         legend_position = NULL) {
```

Comment:
Putting strata to NULL is not our default in visR. If no strata are provided, we have 'Overall'. Is strata argument required? If not filled out we can default to ggplot2 colors right? I dont see the need for a warning.

Reply:
The same is true for font sizes etc ... If the user does not provide this, we can provide a default or take the one of ggplot2

Reply:
Which is exactly what is happening when you call it without any parameters. Them being set to NULL was supposed to be more of a way to expose the options that will be evaluated.
![image](https://user-images.githubusercontent.com/12818242/119939621-71c61400-bf8e-11eb-9eb5-bf3f220aa480.png)

Reply:
Aactually, in `estimate_KM` and `get_tableone` we are setting the `strata` parameter to NULL as default which is treated internally as "overall" in the sense that data is not stratified